taqueria-lib-ligo/common.ts

```diff
@@ -77,3 +78,36 @@ export const configure = (dockerImage: string, dockerImageEnvVar: string) => ({
 });
 
 export type Common = ReturnType<typeof configure>;
+
+function exists(path: string): boolean {
+	try {
+		fs.accessSync(path);
+		return true;
+	} catch {
+		return false;
+	}
+}
+
+function getLigoBinaryFromPath() {
+	const { PATH } = process.env;
+	if (!PATH) {
+		return null;
+	}
+	const paths = PATH.split(delimiter);
+	for (const candidatePath in paths) {
```

Comment:
```suggestion
for (const candidatePath of paths) {
```

Otherwise, `candidatePath` will be the index of the array.